blueprints/adapter-test/qunit-rfc-232-files/tests/unit/__path__/__test__.js

```diff
@@ -0,0 +1,12 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
+module('adapter:<%= dasherizedModuleName %>', '<%= friendlyTestDescription %>', function(hooks) {
```

Comment:
There should not be three arguments here.

Reply:
updated... sorry about that..